ndbench-cockroachdb-plugins/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+dependencies {
+    compile project(':ndbench-api')
+    compile project(':ndbench-core')
+
+    compile 'javax.inject:javax.inject'
+    testCompile group: 'com.netflix.archaius', name: 'archaius2-guice', version: '2.1.11'
+    // https://mvnrepository.com/artifact/org.postgresql/postgresql
+    compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
```

Comment:
Apparently, CockroachDB uses the PostgreSQL wire protocol. Do we want to mention this plugin as Postgres then as it will indicate the wire protocol it uses and is also applicable to CockroachDb and PostgreSQL?

Reply:
I would leave the name as is for now and revisit in the future when we add another plugin that supports PostgreSQL wire protocol, and make this plugin more generic.

Reply:
What are the specific things related to Cockroach in this PR that do not apply to PostgreSQL? CockroachDB works ONLY with the PostgreSQL. More info: https://www.cockroachlabs.com/docs/stable/frequently-asked-questions.html#why-does-cockroachdb-use-the-postgresql-wire-protocol-instead-of-the-mysql-protocol

Reply:
I am not denying we shouldn't ever do the rename. I am just not as comfortable generalizing this plugin beyond cockroach yet. Having said that, I would love this to be a generic PostgreSQL plugin, but I would rather take a shot at it later on, to be confident this is generic enough. If you are curious, there are subtle differences between what PostgreSQL and Cockroach db support. https://www.cockroachlabs.com/docs/stable/porting-postgres.html.